src/Libraries/CoreNodeModels/Input/IntegerSlider.cs

```diff
@@ -22,7 +22,7 @@ namespace CoreNodeModels.Input
     [SupressImportIntoVM]
     [IsDesignScriptCompatible]
     [AlsoKnownAs("DSCoreNodesUI.Input.IntegerSlider")]
-    public class IntegerSlider : SliderBase<int>
+    public class IntegerSlider : SliderBase<long>
```

Comment:
@mjkkirschner is this an API break too?

Reply:
I think so... 
https://docs.microsoft.com/en-us/dotnet/core/compatibility/

I'm not sure if this would be considered a break or judgement call.